.gitignore

```diff
@@ -1,3 +1,6 @@
 /vendor/
 composer.lock
 .php_cs.cache
+
+# IDEs
+.idea
```

Comment:
This is a hard no for two reasons. 
- It belongs in your global gitignore
- Jetbrains actually recommends committing this. 

So no matter the philosophy you have about this directory, you should not add it into a project.

Reply:
Thanks for providing the details. That is all good.